gapis/api/gles/compat.go

```diff
@@ -440,12 +440,18 @@ func compat(ctx context.Context, device *device.Instance) (transform.Transformer
 			src := ""
 
 			if config.UseGlslang {
-				opts := shadertools.Option{
-					IsFragmentShader: shader.Type == GLenum_GL_FRAGMENT_SHADER,
-					IsVertexShader:   shader.Type == GLenum_GL_VERTEX_SHADER,
+				st, err := shader.Type.ShaderType()
+				if err != nil {
+					log.W(ctx, "%v compat: %v", cmd, err)
```

Comment:
That's pretty much what you'll see as the `err` returned from `ShaderType()` will contain the unknown error.